AuthenticatorShared/UI/Vault/VaultItem/AuthenticatorKeyCapture/ManualEntryView.swift

```diff
@@ -117,13 +104,63 @@ struct ManualEntryView: View {
             })
         }
     }
+
+    /// Create a button to trigger an `.addPressed(:)` action.
+    ///
+    /// - Parameter sendToBitwarden: whether this button sends the code to Bitwarden (`true`) or stores
+    ///     it locally (`false`). This is also used to determine the title and accessibility identifier.
+    /// - Returns: the configured `Button`.
+    ///
+    private func addButton(sendToBitwarden: Bool) -> some View {
```

Comment:
🤔 Why not make the button style another argument that's passed in, and eliminate the `addPrimaryButton` and `addSecondary` button functions altogether? Then the calls above could just be `addButton(.primary(), sendToBitwarden: false)` or something along those lines?

Reply:
I tried doing that but you can't do that and return `some View`, it would have to be `any View`. My gut says 100% it should only be one method, but I couldn't find any way to make the compiler agree with me 😆

Reply:
Hah, fair enough